samples/Microsoft.TestPlatform.Protocol/Program.cs

```diff
@@ -24,8 +25,8 @@ public static int Main(string[] args)
             if(args == null || args.Length < 1)
             {
                 Console.WriteLine("Please provide appropriate arguments. Arguments can be passed as following:");
-                Console.WriteLine("Microsoft.TestPlatform.Protocol.exe --testassembly:\"[assemblyPath]\" --operation:\"[RunAll|RunSelected|Discovery|DebugAll|DebugSelected]\" --testadapterpath:\"[path]\"");
-                Console.WriteLine("or Microsoft.TestPlatform.Protocol.exe -a:\"[assemblyPath]\" -o:\"[RunAll|RunSelected|Discovery|DebugAll|DebugSelected]\" -p:\"[path]\" \n");
+                Console.WriteLine("Microsoft.TestPlatform.Protocol.exe --testassembly:\"[assemblyPath]\" --operation:\"[RunAll|RunSelected|Discovery|DebugAll|DebugSelected]\" --framework:Framework45 --testadapterpath:\"[path]\"");
```

Comment:
`Framework45` is not a standard terminology. Can we please make this `.NET Framework, Version=4.5`? I assume later is supported by test platform.